rsmtool/analysis.py

```diff
@@ -368,17 +449,43 @@ def compute_metrics(df,
                     use_scaled_predictions=False,
                     include_second_score=False):
     """
-    Compute the evaluation metrics for the scores
-    in the given data frame. This function compute metrics
-    for all score types. If `include_second_score` is True,
-    then assume that a column called `sc2` containing a second
-    human score is available and use that to compute the human-human
-    evaluation stats and the performance degradation stats.
+    Compute the evaluation metrics for the scores in the given data frame.
+
+    This function compute metrics for all score types.
+    If `include_second_score` is True, then assume that a column called
+    `sc2` containing a second human score is available and use that to
+    compute the human-human evaluation stats and the performance
+    degradation stats.
 
     If `compute_shortened` is set to True, then this function also
     computes a shortened version of the full human-machine metrics data
     frame. See `filter_metrics()` for a description of the default
     columns included in the shortened data frame.
+
+    Parameters
+    ----------
+    df : pandas DataFrame
+        Input data frame
+    compute_shortened : bool, optional
+        Also compute a shortened version of the full
+        metrics data frame, defaults to False.
+    use_scaled_predictions : bool, optional
+        Include scaled predictions for evaluation
```

Comment:
Change to "Use evaluations based on scaled predictions in the shortened version of the metrics data frame"

We always compute full evaluations for both raw and scale  as long as they are supplied in the data.